src/pages/Control.jsx

```diff
@@ -11,50 +11,63 @@ import ErrorMessage from "../molecules/ErrorMessage";
 import GlobalState from "../GlobalState";
 import LoadingIndicator from "../atoms/LoadingIndicator";
 
-import {
-  mockControlData,
-  mockNextControlId,
-  mockInheritedComponentNarratives,
-} from "../mockData/ControlPage";
-
 const ERROR_MESSAGE = "Error loading project control";
 
 export default function Control() {
-  const { id } = useParams();
-
+  const { id, controlId } = useParams();
   const [state, setState] = useContext(GlobalState);
-  const [hasError, setHasError] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
+  const [control, setControl] = useState();
+  const [componentData, setComponentData] = useState();
 
   useEffect(() => {
     if (!state.project || state.project.id !== parseInt(id)) {
       setIsLoading(true);
       RequestService.get(
-        `${Config("backendUrl")}/projects/${id}/`,
+        `${Config("backendUrl")}/projects/${id}/controls/${controlId}/`,
         (response) => {
           setState((state) => ({ ...state, project: response.data }));
```

Comment:
If we're making the api call to the controls endpoint we probably don't want to store the response as "project" in the state- I wonder if we care about using the "global state" at all here?

Reply:
The response contains the project at the top level, which is the only reason we are putting it all in the global state. I agree with the extra data being thrown in here is not ideal, but at this point in time we do not have a better solution. We still want to use global state, as this is what will keep the breadcrumbs displaying correctly. We could address this in the future as an optimization for removing a call to projects on every page if we stored projects in a cookie, but would have to discuss more to make sure we do not have giant cookies slowing down page responses, similar issue would arise if we use local storage, since local storage also has size limits per site.